accessories/alarm.js

```diff
@@ -0,0 +1,102 @@
+const VerisureAccessory = require('./verisure');
+
+class Alarm extends VerisureAccessory {
+  constructor(...args) {
+    super(...args);
+
+    const { alarmCode } = this.platformConfig;
+    this.alarmCode = alarmCode.toString();
+    this.model = 'ALARM';
+    this.name = VerisureAccessory.getUniqueAccessoryName(`Alarm (${this.installation.config.alias})`);
+
+    const { SecuritySystemCurrentState } = this.homebridge.hap.Characteristic;
+    this.armStateMap = {
+      ARMED_AWAY: [SecuritySystemCurrentState.AWAY_ARM],
+      ARMED_HOME: [
+        SecuritySystemCurrentState.STAY_ARM,
+        SecuritySystemCurrentState.NIGHT_ARM,
```

Comment:
@hoefs Correct, both HomeKit arm states "Home" & "Night" are both mapped to the Verisure "Home" state.